canvas_modules/common-canvas/src/common-properties/properties-main/properties-main.jsx

```diff
@@ -47,6 +47,10 @@ class PropertiesMain extends React.Component {
 	constructor(props) {
 		super(props);
 		this.propertiesController = new PropertiesController();
+		// Persist editorSize when resize() is not called
+		if (this.propertiesController.currentEditorSize !== this.props.currentEditorSize) {
```

Comment:
Why check this?  I would expect to only set this if something is set in `this.props.currentEditorSize`